presto-docs/src/main/sphinx/release/release-0.255.rst

```diff
@@ -0,0 +1,43 @@
+=============
+Release 0.255
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix an issue where regular expression functions were not interruptible and could  keep running for a long time after the query they were a part of failed.
+* Add interruption for runaway splits blocked in known situations controlled by ``task.interrupt-runaway-splits-timeout`` property which defaults to ``600s``.
+* Add support for querying non-lowercase table names in Druid connector (:pr:`15920`).
+* Added array_normalize function.
```

Comment:
```
Added ``array_normalize`` function to normalize an array by dividing each element by the p-norm of it.
```
In case people are interested what p-norm is: https://en.wikipedia.org/wiki/Norm_(mathematics)#p-norm

Reply:
Please note that functions have to be written as :func: (Refer some of the prev Release notes PRs for the syntax)